express-delete/server.ts

```diff
@@ -35,11 +35,23 @@ app.get('/api/grades', (req, res) => {
 });
 
 app.delete('/api/grades/:id', (req, res) => {
-  for (const key in grades) {
-    if (Number(req.params.id) === grades[key].id) {
-      delete grades[key];
-      res.sendStatus(204);
+  const id = Number(req.params.id);
+  try {
+    for (const key in grades) {
```

Comment:
This works, but there is no need to iterate through the grades. You can just use, for example, `if (key in grades)` to determine if the `id` exists.